sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -109,6 +134,11 @@ export default defineComponent({
     const studentApplicationOfferingDetails = ref(
       {} as ApplicationOfferingDetails,
     );
+    const assessApplicationOfferingChangeRequestModal = ref(
+      {} as ModalDialog<ApplicationOfferingChangeAssessmentAPIInDTO>,
```

Comment:
should it not be `ModalDialog<ApplicationOfferingChangeAssessmentAPIInDTO | false>`

as we return `resolvePromise(false)` on cancel ?

![image](https://github.com/bcgov/SIMS/assets/54600590/ad2fd0ea-4742-4e1b-987a-a2a70aababb1)